src/uu/tail/src/files.rs

```diff
@@ -0,0 +1,347 @@
+//  * This file is part of the uutils coreutils package.
+//  *
+//  * For the full copyright and license information, please view the LICENSE
+//  * file that was distributed with this source code.
+
+// spell-checker:ignore (ToDO) tailable seekable
+
+use crate::args::Settings;
+use crate::text;
+use core::option::Option;
+use core::option::Option::{None, Some};
+use core::result::Result::{Err, Ok};
```

Comment:
All of these `Option`/`Result` things are in the prelude so don't need to be imported right?